### ISSUE TITLE
multiple passphrase asks for one command

### ISSUE THREAD

**Issue by mr-tron** [sonm-io/core#933]
![image](https://user-images.githubusercontent.com/702560/40878733-17a109c4-668d-11e8-9703-5d1cc474f319.png)
I think sonmcli can remeber passphrase during one command execution.

**Comment by mr-tron** [comment]
```
user@work:~/apps/sonm/cli$  ./sonmcli version
sonmcli v0.4.0-f31ec65 (linux/amd64/go1.10)
user@work:~/apps/sonm/cli$ cat ~/.sonm/cli.yaml 
ethereum:
  pass_phrase: ""
  key_store: ""
output_format: simple
worker_eth_addr: ""
```

**Comment by 3Hren** [comment]
Hi!

We have a plan (during this month) to improve the CLI by using default system keychain. This should fix this issue.

**Comment by sshaman1101** [comment]
fixed in [v0.4.1](https://github.com/sonm-io/core/releases/tag/v0.4.1) (#1141), closing PR for now.
